proto/hummock.proto

```diff
@@ -227,11 +227,13 @@ message HummockPinnedSnapshot {
   uint64 minimal_pinned_snapshot = 2;
 }
 
-message GetNewTableIdRequest {}
+message GetNewSstIdsRequest {
+  uint32 number = 1;
+}
 
-message GetNewTableIdResponse {
+message GetNewSstIdsResponse {
   common.Status status = 1;
-  uint64 table_id = 2;
+  repeated uint64 table_id = 2;
```

Comment:
the returned table_id here is guaranteed to be continous. can we use begin_id and batch_num in the response instead of a vec?

Reply:
fixed